api/src/controllers/staff.ts

```diff
@@ -0,0 +1,60 @@
+import services from "../services";
+import { Request, Response } from "express";
+import response from "../utils/response";
+import { ICreateStaff } from "../models/create-request";
+import { IQueryStaff } from "../models/query-request";
+import responseMsg from "../const/response-msg";
+
+const create = async (req: Request, res: Response) => {
+  const staff: ICreateStaff = req.body as ICreateStaff;
+  const [data, error] = await services.staff.create(staff);
+  if (error) {
```

Comment:
hệ thống của mình chỉ cần trả các lỗi sau:
- 200: success
- 400: bad request
- 401: unauthorized
- 404: not found

như dưới sẽ là

```js
if (error) {
  return response.r400(res, {}, err.message)
}
response.r200(res, data)
```

Reply:
done

Reply:
cái này chỉ cần như code sample thôi, dư cái 404 nhé

Reply:
done